projects/ngxpert/hot-toast/src/lib/components/hot-toast-container/hot-toast-container.component.ts

```diff
@@ -190,7 +190,14 @@ export class HotToastContainerComponent {
 
   closeToast(id?: string) {
     if (id) {
-      const comp = this.hotToastComponentList.find((item) => item.toast.id === id);
+      const comp = this.hotToastComponentList.find((item) => {
+        // Type guard to check if Toast has an 'id' property
```

Comment:
What is need of this? `Toast` will always has an `id`.

Reply:
TypeScript raised an error, so I added a check to resolve it